### PR TITLE
update links for community meeting, gitter

### PR DESCRIPTION
**Pull request by suhuruli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,9 +70,9 @@ One of the easiest ways to contribute is to participate in discussions. There ar
...
 | Item        | Value  |
 |---------------------|---|
-| Mailing List | Coming soon! |
-| Slack Channel       | Coming soon! |
-| Community meeting | Every other Tuesday at 10:30AM PST. Zoom meeting link: https://zoom.us/j/623691799?pwd=ZWc4SHFNdWpRUVVNYkdJWE9zVHpjZz09  |
-| Meeting Notes       | https://docs.google.com/document/d/1nqdFEyULekyksFHtFvgvFAYE-0AMHKoS3RMnaKsarjs/edit?usp=sharing |
+| Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
+| Meeting Information | Bi-weekly (Starting October 23rd), Tuesdays 10:30AM PST  |
+| Meeting Link | https://zoom.us/j/271516061  |
+| IM Channel       | https://gitter.im/oam-dev/ |
```
Comment (xiang90) [inline README.md]: why do we remove the meeting notes doc?

**Comment by vturecek** [comment]
Oh, I just opened the same PR! I'll close mine, let's use this one.

File: README.md [hunk 2]
```diff
@@ -70,9 +70,9 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 
 | Item        | Value  |
 |---------------------|---|
-| Mailing List | Coming soon! |
-| Slack Channel       | Coming soon! |
-| Community meeting | Every other Tuesday at 10:30AM PST. Zoom meeting link: https://zoom.us/j/623691799?pwd=ZWc4SHFNdWpRUVVNYkdJWE9zVHpjZz09  |
-| Meeting Notes       | https://docs.google.com/document/d/1nqdFEyULekyksFHtFvgvFAYE-0AMHKoS3RMnaKsarjs/edit?usp=sharing |
+| Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
+| Meeting Information | Bi-weekly (Starting October 23rd), Tuesdays 10:30AM PST  |
```
Comment (hongchaodeng) [inline README.md]:
```suggestion
| Meeting Information | Bi-weekly (Starting Oct 23, 2019), Tuesdays 10:30AM PST  |
```

File: README.md (hunk 1 above)
Comment (technosophos) [reply README.md]: See #192

Comment (suhuruli) [reply README.md]: I added it back in, I must have left it out by accident

**Comment by vturecek** [comment]
Looks like my PR got merged with the exact same changes, so we can probably just close this one

**Pull request closed without merging**